src/constants/skills.js

```diff
@@ -233,9 +232,10 @@ const icons = {
     "https://raw.githubusercontent.com/devicons/devicon/master/icons/backbonejs/backbonejs-original-wordmark.svg",
   bootstrap:
     "https://raw.githubusercontent.com/devicons/devicon/master/icons/bootstrap/bootstrap-plain-wordmark.svg",
-  c: "https://raw.githubusercontent.com/devicons/devicon/master/icons/c/c-original.svg",
+  c:
+    "https://raw.githubusercontent.com/devicons/devicon/master/icons/c/c-original.svg",
```

Comment:
Why does this one get 2 lines?

Reply:
This is because of my code formatter if you request I will remove these lines.

Reply:
Seems a little weird, but it was inconsistent before anyhow, and I guess I don't really care.